src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -2814,6 +2819,54 @@ public void testSideEffect() {
     tracer.setExpected("sideEffect", "sleep PT1S", "executeActivity customActivity1");
   }
 
+  private static final Map<String, Queue<Long>> mutableSideEffectValue =
```

Comment:
I don't think we will need the dictionary. Single queue is enough?

Reply:
It is because the tests are executed in parallel. We need to fix all the other places where globals are used by workflow code.